sdk/src/wallet/operations/balance.rs

```diff
@@ -73,14 +64,27 @@ where
                 Output::Account(account) => {
                     // Add amount
                     balance.base_coin.total += account.amount();
+                    // Add stored mana
+                    balance.mana.total.stored += protocol_parameters.mana_with_decay(
+                        account.mana(),
+                        output_id.transaction_id().slot_index(),
+                        slot_index,
+                    )?;
+                    // Add potential mana
+                    balance.mana.total.potential += protocol_parameters.generate_mana_with_decay(
+                        account.amount().saturating_sub(storage_cost),
+                        output_id.transaction_id().slot_index(),
+                        slot_index,
+                    )?;
```

Comment:
Can you not just call `decayed_mana` on the output then?

Reply:
yeah right I forgot those methods were added

Reply:
done